messaginginapp/src/main/java/io/customer/messaginginapp/ModuleMessagingInApp.kt

```diff
@@ -1,30 +1,44 @@
 package io.customer.messaginginapp
 
 import android.app.Application
+import androidx.annotation.VisibleForTesting
+import io.customer.base.internal.InternalCustomerIOApi
 import io.customer.messaginginapp.di.gistProvider
 import io.customer.messaginginapp.hook.ModuleInAppHookProvider
 import io.customer.sdk.CustomerIO
+import io.customer.sdk.CustomerIOConfig
 import io.customer.sdk.data.request.MetricEvent
 import io.customer.sdk.di.CustomerIOComponent
 import io.customer.sdk.hooks.HookModule
 import io.customer.sdk.hooks.HooksManager
 import io.customer.sdk.module.CustomerIOModule
 import io.customer.sdk.repository.TrackRepository
 
-class ModuleMessagingInApp internal constructor(
+@VisibleForTesting
+class ModuleMessagingInApp @InternalCustomerIOApi internal constructor(
     override val moduleConfig: MessagingInAppModuleConfig = MessagingInAppModuleConfig.default(),
-    private val overrideDiGraph: CustomerIOComponent?,
-    private val organizationId: String
+    private val overrideDiGraph: CustomerIOComponent?
 ) : CustomerIOModule<MessagingInAppModuleConfig> {
 
     @JvmOverloads
+    @Deprecated(
+        "organizationId no longer being used and will be removed in future",
```

Comment:
May be for only me, but `being used` is slightly confusing here I think. I would suggest it to update as:

```suggestion
        "organizationId no longer required and will be removed in future",
```

Reply:
I followed the same one as we are using in `iOS` but this does make more sense :D